modules/local/detect_ncrna.nf

```diff
@@ -28,9 +30,21 @@ process DETECT_NCRNA {
     # De-overlap #
     grep -v " = " overlapped_${fasta.baseName} > ${meta.prefix}.ncrna.deoverlap.tbl
 
+    echo "Parsing final results..."
+    parse_rRNA-bacteria.py \
```

Comment:
so.. we were running cmscan and cmsearch.. with no need. We need to propagate this to other pipelines, mOTUs, main MGnify ones.

Reply:
Yes, fixing this in the genomes pipeline is on my to do list next and we need to keep this in mind for the future.